app/assets/stylesheets/styles.css.scss

```diff
@@ -0,0 +1,45 @@
+
+body {
+  background: transparent url("texture.png");
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.